src/main/java/school/lemon/changerequest/java/Matrix.java

```diff
@@ -0,0 +1,126 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 11.12.16.
+ */
+public class Matrix {
+
+    private int rows;
+    private int columns;
+    private double[][] matrix;
+
+
+    public Matrix(int rows, int columns) {
+        if ((rows < 1) || (columns < 1)) {
+            System.out.println("Cannot create a matrix smaller that 1x1");
+            System.exit(0);
+        } else {
+            this.rows = rows;
+            this.columns = columns;
+            this.matrix = new double[rows][columns];
+        }
+    }
+
+    public Matrix(double[][] matrix) {
+
+        for (int i = 1; i < matrix.length; i++) {
+            if (matrix[i].length != matrix[0].length)
+                throw new RuntimeException("Illegal vector dimensions"); //verification that all rows has the same columns number
+        }
+        this.rows = matrix.length;
+        this.columns = matrix[0].length;
+        this.matrix = new double[rows][columns];
+        for (int i = 0; i < rows; i++) {
+            for (int j = 0; j < columns; j++) {
+                this.matrix[i][j] = matrix[i][j];
+            }
+        }
+    }
+
+    public boolean checkSize(Matrix matrixB) {
+        if ((this.rows != matrixB.rows) || (this.columns != matrixB.columns))
+            throw new RuntimeException("Illegal vector dimensions.");
```

Comment:
It's not correct type of exception. Think about the most precise exception name.